jujugui/static/gui/src/app/app.js

```diff
@@ -1774,7 +1784,13 @@ YUI.add('juju-gui', function(Y) {
           this._renderMachineView.bind(this),
           this._clearMachineView.bind(this)],
         ['gui.inspector',
-          this._renderInspector.bind(this)]
+          this._renderInspector.bind(this)],
+        ['gui.deploy',
+          this._renderDeployment.bind(this),
+          this._clearDeployment.bind(this)],
+        ['deploy',
```

Comment:
We shoudln't have a base 'deploy' state required here - I also don't see where you've changed any of the changeState's to add a non `gui` state 'deploy'. Was this left in?